qqmlsortfilterproxymodel_plugin.cpp

```diff
@@ -0,0 +1,31 @@
+#include "qqmlsortfilterproxymodel_plugin.h"
+
+#include <qqml.h>
+
+#include "qqmlsortfilterproxymodel.h"
+#include "filter.h"
+#include "sorter.h"
+
+using namespace qqsfpm;
+
+void QQmlsortfilterproxymodelPlugin::registerTypes(const char *uri)
+{   
+    //Filter
+    qmlRegisterUncreatableType<Filter>("SortFilterProxyModel", 0, 2, "Filter", "Filter is an abstract class");
```

Comment:
None of these fit our code guidelines for line breaks between parameters (*don't kill me*). However, I would actually prefer keeping these as-is *if they are consistent with the style used in the rest of this 3rd-party code*. So I guess my comment would be: please make sure the code style used in this file is consistent with the rest of the code style in this repository (if it isn't already).